models/localized-string/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@commercetools-test-data/localized-string",
+  "version": "0.1.0",
+  "description": "Data model for LocalizesString type",
+  "bugs": "https://github.com/commercetools/test-data/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/test-data.git",
+    "directory": "models/localized-string"
+  },
+  "keywords": [
+    "javascript",
+    "test-data"
+  ],
+  "license": "MIT",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./dist/localized-string-index.cjs.js",
+  "module": "./dist/localized-string-index.es.js",
+  "typings": "./dist/typings/index.d.ts",
+  "types": "./dist/typings/index.d.ts",
```

Comment:
Just wondering what's the difference between `typings` and `types`?

Reply:
Somewhat a convention I think. At least for us. Types are external. Typings are what we build and expose as types to others.